tools/cli/go-whisk-cli/commands/util.go

```diff
@@ -427,17 +427,31 @@ func printRuleSummary(rule *whisk.Rule) {
 }
 
 func printEntitySummary(entityType string, fullName string, description string, params string) {
-    if len(description) > 0 {
+    if len(description) > 0 && len(params) > 0 {
         fmt.Fprintf(color.Output, "%s %s: %s\n", boldString(entityType), fullName, description)
+    } else if len(params) > 0 {
+        descriptionFromParams := buildParamString(params)
+        fmt.Fprintf(color.Output, "%s %s: %s\n", boldString(entityType), fullName, descriptionFromParams)
     } else {
         fmt.Fprintf(color.Output, "%s %s\n", boldString(entityType), fullName)
+        fmt.Fprintf(color.Output, "   (%s: %s has no annotations)\n", boldString("Warning"), fullName)
```

Comment:
When you say "externalized", I'm assuming you mean put the text into a variable and just call the variable in the print statement?